### PR TITLE
Modified BufferPool.java

### PR DESCRIPTION
**Pull request by Rithesh17**
I have implementer the InsertTuple and the DeleteTuple methods. The code was pretty easy and I didn't take much time for it.

### CODE REVIEW

**Review by manishsbhoopalam — APPROVED** [review]

**Review by Ram-Aditya — APPROVED** [review]

**Review by Priiyam — APPROVED** [review]

**Comment by sbshah97** [comment]
Why has this Pull Request not been merged?

**Comment by Ram-Aditya** [comment]
@salman-bhai  We thought we needed to wait for your approval as we had requested a review from your side.
Sorry about that!We `ll go ahead with the merge now then.

**Comment by Ram-Aditya** [comment]
Closing this pull request as updated one has been put up already.

**Pull request closed without merging**
